1470.js

```diff
@@ -0,0 +1,14 @@
+/**
+ * @param {number[]} nums
+ * @param {number} n
+ * @return {number[]}
+ */
+var shuffle = function (nums, n) {
+  const result = [];
+  for (let i = 0; i < n; i++) {
+    result.push(nums[i]);
+    result.push(nums[n + i]);
```

Comment:
You can push multiple elements in one line 👍 
```suggestion
    result.push(nums[i], nums[n + i]);
```